### PR TITLE
verb histogram colors and complexity

### PR DESCRIPTION
**Pull request by eelegiap**
- Updates to verb histogram tool: new coloring scheme for the constructions + option for simpler complexity

### CODE REVIEW

File: README 2.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+![Coverage Status](./coverage.svg)
```
Comment (arthurian) [inline README 2.md:1]: @eelegiap It looks like you may have accidentally committed this copy of the README. Did you mean to do that?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/Harvard-ATG/visualizing_russian_tools.svg?branch=master)](https://travis-ci.org/Harvard-ATG/visualizing_russian_tools)
```
Comment (arthurian) [inline README.md:1]: This looks like an old version of the `README.md` that's trying to be merged in. Do you have the latest version in your repo?

**Review by arthurian — CHANGES_REQUESTED** [review]
@eelegiap This looks good except for the two README changes. Can you fix those two things and then we can merge? Thanks!

**Comment by eelegiap** [comment]
@arthurian I've fixed the problems with the README, thanks for pointing that out!

**Comment by arthurian** [comment]
Thanks @eelegiap! :+1:

**Pull request merged**